### DISCUSSION TITLE
New-PnPUPABulkImportJob not syncing for external GEO's in a multi-geo tenant.

### DISCUSSION THREAD

**Discussion by OgHoomans** [pnp/powershell#4172] [Q&A]
Hello Everyone,

I’m seeking assistance with a script I’ve developed to synchronize Active Directory (AD) custom attributes to SharePoint Online user profiles. The script functions correctly for syncing properties to user profiles within our default NAM (North America) region. However, I encounter issues when attempting to sync to user profiles in other geo-regions.

Here’s a brief overview of the process:

File Upload: I upload a file containing user attributes to a SharePoint site in the external geo-region.
Job Execution: I connect to the external geo’s admin center to initiate the sync job.
The job starts as expected, but after some time, the Get-PnPUPABulkImportStatus command results in an error with the message "Value cannot be null". I assume this error indicates that the job may have completed, but I am unable to confirm the final status or success of the sync. I have checked the user profiles in SharePoint, but their custom attributes remain empty.

Could anyone advise on the following:

How can I verify the final status of an import job in a multi-geo environment?
Are there specific steps or additional troubleshooting I should consider to ensure that custom attributes are correctly updated?
Any insights or suggestions would be greatly appreciated.

**Comment by Id10tSupport** [comment]
In my experience, any custom attributes you have in the default region (NAM most likely) do not automatically exist in the UPA in the geo-located region UPAs, so you may need to manually add them.  Additionally, be sure to use the -JobId ID number when running Get-PnPUPABulkImportStatus as it may be attempting to look for the most recent Upload job run on the default (NAM) region as well.
